src/middleware/audioMiddleware.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+/**
+ * This middleware is responsible for forwarding audio actions to the mediaservice
```

Comment:
And in my opinion it should be the ONLY one that is allowed to access the MediaService instance.

Reply:
✅ done